ec/acc/testdata/extension_bundle_download.tf

```diff
@@ -0,0 +1,6 @@
+resource "ec_extension" "my_extension" {
+  name           = "%s"
+  version        = "*"
+  extension_type = "bundle"
```

Comment:
This test will download a `plugin` not a bundle, needs to be changed:
```suggestion
  extension_type = "plugin"
```

Reply:
fixed by d340fa3